<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupervisedChartWidget</class>
 <widget class="QWidget" name="SupervisedChartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>103</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="labelXAxis">
       <property name="text">
        <string>X-axis input:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxXAxis</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelYAxis">
       <property name="text">
        <string>Y-axis input:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxYAxis</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxXAxis"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxYAxis"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelOutput">
       <property name="text">
        <string>Output:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxOutput</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxOutput"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxOutput</tabstop>
  <tabstop>comboBoxXAxis</tabstop>
  <tabstop>comboBoxYAxis</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
